### PR TITLE
ci: Auto-approve and merge dependabot PRs

### PR DESCRIPTION
**Pull request by terrytangyuan**
Signed-off-by: terrytangyuan <terrytangyuan@gmail.com>

### CODE REVIEW

**Comment by terrytangyuan** [comment]
This will auto-merge dependabot PRs (when CI builds pass) but one issue is that CI will pass anyway if there's no review requested. Should we remove `review_requested`  in `ci.yml`?

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ on:
       - '**.go'
       - 'Makefile'
   pull_request:
-    types: [review_requested, ready_for_review]
+    types: [ready_for_review]
     paths:
       - '.github/workflows/**'
       - '**.go'
```
Comment (gaocegege) [inline .github/workflows/CI.yml:15]: We should add go.mod and go.sum to trigger tests when the bot files PRs.

**Comment by gaocegege** [comment]
> Should we remove review_requested from ci.yml? I remember it's only used to save some build time on GA. We should have enough build time now since the project is open sourced.

Sure, we can remove it.

File: .github/workflows/CI.yml (hunk 1 above)
Comment (terrytangyuan) [reply .github/workflows/CI.yml:16]: Added.

**Review by gaocegege — APPROVED** [review]

**Pull request merged**